includes/Modules/Ads.php

```diff
@@ -85,7 +93,64 @@ protected function setup_assets() {
 					),
 				)
 			),
+			new Script_Data(
+				'googlesitekit-ads-pax-config',
+				array(
+					'global'        => '_googlesitekitPAXConfig',
+					'data_callback' => function() {
+						if ( ! current_user_can( Permissions::VIEW_DASHBOARD ) ) {
+							return array();
+						}
+
+						return array(
+							'authAccess'      => array(
+								'oauthTokenAccess' => array(
+									'token' => (string) $this->authentication->get_oauth_client()->get_access_token(),
+								),
+							),
+							'locale'          => mb_substr( $this->context->get_locale( 'user' ), 0, 2 ),
+							'debuggingConfig' => array(
+								'env' => 'PROD',
+							),
+						);
+					},
+				)
+			),
 		);
+
+		if ( current_user_can( Permissions::VIEW_DASHBOARD ) && $this->is_connected() ) {
+			$assets[] = new Script(
+				'googlesitekit-ads-pax-integrator',
+				array(
+					'src'          => 'https://www.gstatic.com/pax/dev/pax_integrator.js',
+					'dependencies' => array(
+						'googlesitekit-ads-pax-config',
+						'googlesitekit-modules-data',
+					),
```

Comment:
Please also set the script `execution` to `async` since it isn't needed to be blocking.